CMakeLists.txt

```diff
@@ -1500,10 +1560,20 @@ add_library(${CoreLibName} ${CoreLinkType}
 	$<TARGET_OBJECTS:GPU>
 	Globals.h
 	git-version.cpp)
-target_link_libraries(${CoreLibName} Common native kirk cityhash sfmt19937 xbrz xxhash
+
+if(ANDROID)
+  set(CoreExtraLibs ${CoreExtraLibs} android)
+  if(X86_64)
+    set(CoreExtraLibs -Wl,--gc-sections -Wl,--exclude-libs,ALL)
+  endif()
+endif()	
+
+target_link_libraries(${CoreLibName} Common native kirk cityhash sfmt19937 xbrz xxhash glslang
```

Comment:
Hmm, should `glslang` only be on Vulkan?  Probably it's safe to add it everywhere?

-[Unknown]